flink-test-utils-parent/flink-test-utils/src/test/java/org/apache/flink/state/benchmark/StateBackendBenchmarkUtils.java

```diff
@@ -216,7 +225,7 @@ private static HeapKeyedStateBackend<Long> createHeapKeyedStateBackend(File root
         return backendBuilder.build();
     }
 
-    private static File prepareDirectory(String prefix, File parentDir) throws IOException {
+    public static File prepareDirectory(String prefix, File parentDir) throws IOException {
```

Comment:
Why we need to make this method public?

Reply:
Like I said in description:

> Also by the way, the StateBackendBenchmarkUtils#prepareDirectory is made public for the de-duplication of code in RescalingBenchmarkBase in state benchmarks.

You may see the [`RescalingBenchmarkBase#prepareDirectory` ](https://github.com/apache/flink-benchmarks/blob/e0d922cb30c3cee827cec05981befae28ec4daa3/src/main/java/org/apache/flink/state/benchmark/RescalingBenchmarkBase.java#L66).  I want to do some code abstraction there.